sunpy/map/map_factory.py

```diff
@@ -4,12 +4,14 @@
 import glob
 from collections import OrderedDict
 import warnings
-
 import numpy as np
 import astropy.io.fits
+from astropy.wcs import WCS
+from astropy.io import fits
 
 import sunpy
-from sunpy.map.mapbase import GenericMap
+from sunpy.data.sample import AIA_171_IMAGE
```

Comment:
This import needs to be removed.

Reply:
The one which imports sample data?

Reply:
Exactly, it doesn't do anything.

Reply:
But if I remove this 'from sunpy.data.sample import AIA_171_IMAGE'
I won't be able to use AIA_171_IMAGE anywhere

Reply:
And I am getting a timout error so is this a fault from the server side?

Reply:
But you only need to use that image in the example. The imports at the top of a file are for use in the  normal code and not documentation strings. 

Yeah, those timeouts are from the server side. You can ignore those.

Reply:
So if I remove that import statement am I good to go with my first PR?